aspnet/fundamentals/servers.rst

```diff
@@ -13,39 +13,44 @@ ASP.NET Core is completely decoupled from the web server environment that hosts
+The default web host for ASP.NET apps developed using Visual Studio is IIS Express functioning as a reverse proxy server for Kestrel. The "Microsoft.AspNetCore.Server.Kestrel" and "Microsoft.AspNetCore.Server.IISIntegration" dependencies are included in *project.json* by default, even with the Empty web site template. Visual Studio provides support for multiple profiles. In addition to the default profile for running in IIS Express, the templates include a second profile that executes the app directly relying on Kestrel for self-hosting. You can manage these profiles and their settings in the **Debug** tab of your web application project's Properties menu or from the *launchSettings.json* file.
 
 .. image:: /fundamentals/servers/_static/serverdemo-properties.png
 
-.. note:: IIS doesn't support commands. Visual Studio launches IIS Express and loads the application with the selected profile.
+.. note:: IIS supports proxying requests to Kestrel but **not** WebListener.
 
-The sample project for this article is configured to support each server option in the *project.json* file:
+The sample project's *project.json* file includes the dependencies and tools required to support each server:
 
 .. literalinclude:: servers/sample/ServersDemo/src/ServersDemo/project.json
-  :lines: 1-17
-  :emphasize-lines: 12-13
+  :lines: 1-18,43-46
+  :emphasize-lines: 5-7,16,20
   :linenos:
   :language: javascript
   :caption: project.json (truncated)
 
-The ``run`` command will launch the application from the ``void main`` method. The ``run`` command configures and starts an instance of ``Kestrel``.
+:dn:method:`~Microsoft.AspNetCore.Hosting.WebHostBuilderKestrelExtensions.UseKestrel` and :dn:method:`~Microsoft.AspNetCore.Hosting.WebHostBuilderWebListenerExtensions.UseWebListener` both have an overload taking an options configuration callback that can be used for server-specific configuration. For instance, WebListener exposes ``AuthenticationManager`` that can be used to configure the server's authentication. Configuration can easily be driven by JSON text files, environment variables, command line arguments and more with the help of ASP.NET Core's :doc:`configuration` facilities.
+
+``dotnet run`` will launch the application from the ``Main`` method. Kestrel is selected by default in the sample project. The sample is programmed so WebListener can be selected instead by passing ``--server WebListener`` as a command line argument. The sample explicitly reads the ``--server`` command line argument in ``Program.Main`` to determine whether to call :dn:method:`~Microsoft.AspNetCore.Hosting.WebHostBuilderKestrelExtensions.UseKestrel` or :dn:method:`~Microsoft.AspNetCore.Hosting.WebHostBuilderWebListenerExtensions.UseWebListener`. The ``--server`` command line flag is **not** interpreted by the ASP.NET Core framework to have any special meaning.
```

Comment:
"The Main method is the entry point for the application." Remove the part about dotnet run, there are several other common ways to launch the app.